docs/chain-details/chain-id.md

```diff
@@ -0,0 +1,52 @@
+# Chain ID and Address Format
+
+## Chain ID
+
+Crypto.com Chain has different Chain ID to distinguish between _devnet_, _testnet_ and _mainnet_. When running the Crypto.com Chain in your local environment, you will also need to decide your own Chain ID.
+
+For example, our testnet Chain ID is `testnet-croeseid-1`.
+
+## Address prefix
+
+[BIP-0173](https://github.com/satoshilabs/slips/blob/master/slip-0173.md) defines a new format for segregated witness output addresses that contains a human-readable part that identifies the coin type. Crypto.com chain has different address prefixes for its corresponding network types, these prefixes are:
+
+| Mainnet | Testnet | Devnet |
+| ------- | ------- | ------ |
+| `cro`   | `tcro`  | `dcro` |
+
+Crypto.com chain uses the Bech32 address format wherever users must handle binary data. Bech32 encoding provides robust integrity checks on data and the human readable part(HRP) that provides contextual hints that can assist UI developers with providing informative error messages. Specifically, we have the following HRP prefix for different addresses types in the mainnet:
+
+|                    | Address bech32 Prefix | Pubkey bech32 Prefix |
```

Comment:
pubkey bech32 representation may be removed in the SDK release

Reply:
Yes, I will remove the `Pubkey bech32 Prefix` here and keep an eye on https://github.com/cosmos/cosmos-sdk/issues/7447.